optimum/intel/openvino/quantization.py

```diff
@@ -220,18 +229,20 @@ def _onnx_export(
             )
             model.enable_dynamic_graph_building()
 
-    def _set_feature(self):
-        if self.feature is None:
-            self.feature = HfApi().model_info(self.model.config._name_or_path).pipeline_tag
-            if self.feature in ["sentiment-analysis", "text-classification", "zero-shot-classification"]:
-                self.feature = "sequence-classification"
-            elif self.feature in ["feature-extraction", "fill-mask"]:
-                self.feature = "default"
-            elif self.feature == "text-generation":
-                self.feature = "causal-lm"
-            elif self.feature is None:
-                raise ValueError("The feature could not be extracted and needs to be specified for the ONNX export.")
-        if self.feature in ["seq2seq-lm", "translation", "summarization"]:
+    def _set_task(self):
+        if self.task is None:
+            self.task = HfApi().model_info(self.model.config._name_or_path).pipeline_tag
+            if self.task in ["sentiment-analysis", "text-classification", "zero-shot-classification"]:
+                self.task = "sequence-classification"
+            elif self.task in ["feature-extraction", "fill-mask"]:
+                self.task = "default"
+            elif self.task == "text-generation":
+                self.task = "causal-lm"
+            elif self.task is None:
+                raise ValueError(
+                    "The task defining the model topology could not be extracted and needs to be specified for the ONNX export."
```

Comment:
```suggestion
                    "The task defining the model topology could not be extracted and needs to be specified for the ONNX export. For example: `OVQuantizer.from_pretrained(model, task='text-classification')`"
```

No preferences on specific phrasing, but I think it would be useful to give an example of how to specify this task.